internal/nginx/config/http/config.go

```diff
@@ -74,15 +79,15 @@ type SplitClientDistribution struct {
 	Value   string
 }
 
-// Map defines an NGINX map.
-type Map struct {
+// NGINXMap defines an NGINX map.
+type NGINXMap struct {
```

Comment:
since the http package deals with NGINX http-related configuration and the map is an NGINX directive, I don't thnik we need to prefix it with NGINX.